website/static/js/project-organizer.js

```diff
@@ -364,10 +375,14 @@ var tbOptions = {
                     }
                     tb.filter(event);
                 },
-                onchange: m.withAttr('value', tb.filterText),
+                onchange: function() {
+                    m.withAttr('value', tb.filterText);
```

Comment:
I do not understand what the `m.withAttr('value', tb.filterText);` line is doing.  If I delete it completely nothing seems to change.  Can you explain what it is for?

Search is happening `onkeyup` and if inside the onchange I log `tb.filterText()` and `this.value`, both are the same thing.  If I type the word "carrot" in the search bar and press enter, both this.value and tb.filterText() are equal to 'carrot' without the  `m.withAttr('value', tb.filterText);` line.